src/ui/ListItem/ListItem.js

```diff
@@ -17,7 +17,7 @@ const Content = styled(({ underline, ...rest }) => <Box {...rest} />)`
 `;
 
 const ListItem = ({ children, icon, underline }) => (
-  <Box as="li" vAlignContent="center">
```

Comment:
We want this to be an `li` to be proper syntax. Maybe try adding `as="span"` to `Content` (ie. `const Content = styled(({ underline, ...rest }) => <Box as="span" {...rest} />)` and to the icon `<Box as="span" width={40}>`